pkg/labels/labels.go

```diff
@@ -301,16 +301,14 @@ func Compare(a, b Labels) int {
 		if a[i].Name != b[i].Name {
 			if a[i].Name < b[i].Name {
 				return -1
-			} else {
-				return 1
 			}
+			return 1
```

Comment:
Should we be consistent with https://github.com/prometheus/prometheus/pull/6845

Reply:
I don't understand #6845. How can that be by design?

Reply:
This was left as it until now as it is easier to reason about. (that was @brian-brazil 's call at the time)

Reply:
But yeah, if we need to chose between this and addinfg a comment to disable the check, let's pick this solution.